src/Microsoft.AspNetCore.Mvc.RazorPages/PageBase.cs

```diff
@@ -149,7 +149,6 @@ public override void EndContext()
         /// Creates a <see cref="BadRequestResult"/> that produces a <see cref="StatusCodes.Status400BadRequest"/> response.
         /// </summary>
         /// <returns>The created <see cref="BadRequestResult"/> for the response.</returns>
-        [NonAction]
```

Comment:
@Eilon is removing an attribute a breaking change?

Reply:
In this case it has no real effect, right? I think that's fine. Only in the craziest case where someone derived from `PageBase` and used the type **as a POCO controller**, would anything behave differently. Too bad for those people.